client/src/main/scala/org/http4s/client/PoolManager.scala

```diff
@@ -30,7 +33,7 @@ private final class PoolManager[A <: Connection](builder: ConnectionBuilder[A],
   private def createConnection(key: RequestKey, callback: Callback[A]): Unit = {
     if (allocated < maxTotal) {
       allocated += 1
-      Task.fork(builder(key)).runAsync {
+      Task.fork(builder(key))(es).runAsync {
```

Comment:
0.12.2 note: this one is sadder.  We could create an internal "default strategy."  What you've done is the right approach in the long run.